UPGRADE.md

```diff
@@ -2672,7 +2672,7 @@ you should run `kong migrations up --force` instead.
 The following commands should be used to prepare a new 1.1 cluster from a fresh datastore:
 
 ```
-$ kong migrations bootstrap [-c config]
+$ kon the the g migrations bootstrap [-c config]
```

Comment:
Is it ok?

Reply:
ah wait i guess not its a mistake ill be setting up another PR in a min